packages/default/scss/pdf-viewer/_layout.scss

```diff
@@ -149,6 +149,7 @@
 
                 section {
                     position: absolute;
+                    pointer-events: auto;
```

Comment:
Do we need to explicitly add `pointer-events: auto`? The element will behave as it would if the pointer-events property were not specified. 🤔